### PR TITLE
CONCF-696 | PHP Fatal Error: Call to a member function getTitle() on null in /extensions/wikia/MercuryApi/MercuryApiController.class.php on line 100

### PR DESCRIPTION
**Pull request by tomasznapieralski**

### CODE REVIEW

**Comment by tomasznapieralski** [comment]
Please CR @Wikia/x-wing

File: extensions/wikia/MercuryApi/MercuryApiController.class.php [hunk 1]
```diff
@@ -97,6 +97,9 @@ private function getArticleDetails( $articleId ) {
 	 */
 	private function getArticleDescription( $articleId, $descLength = 100 ) {
 		$article = Article::newFromID( $articleId );
+		if ($article === null) {
```
Comment (macbre) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php]: Let's use `$article instanceof Article` instead.

Comment (hakubo) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php]: +1

also add some spacing around for better readability

**Comment by bognix** [comment]
LGTM

**Comment by Warkot** [comment]
LGTM :shipit:

File: extensions/wikia/MercuryApi/MercuryApiController.class.php [hunk 2]
```diff
@@ -97,6 +97,11 @@ private function getArticleDetails( $articleId ) {
 	 */
 	private function getArticleDescription( $articleId, $descLength = 100 ) {
 		$article = Article::newFromID( $articleId );
+
+		if ( $article instanceof Article ) {
```
Comment (Warkot) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php]: You forgot about a `!`. LMFTFY :wrench:

**Pull request merged**